src/main/java/com/answerdigital/answerking/response/CategoryResponse.java

```diff
@@ -22,16 +22,12 @@ public class CategoryResponse {
 
     private String description;
 
-    @JsonProperty(access = JsonProperty.Access.READ_ONLY)
```

Comment:
In API spec we have these fields with read-only status, just wondering if removing this annotation here will cause that we won't be compliant with the spec?